frontend/package.json

```diff
@@ -23,16 +24,15 @@
     "react-dom": "17.0.2",
     "react-dropzone": "12.1.0",
     "react-router-dom": "6.3.0",
-    "react-scripts": "5.0.1",
     "react-tsparticles": "1.43.1",
     "uuid": "8.3.2",
+    "vite": "^3.0.4",
+    "vite-tsconfig-paths": "^3.5.0",
     "web-vitals": "2.1.4"
   },
   "scripts": {
-    "start": "ESLINT_NO_DEV_ERRORS='true' react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "start": "ESLINT_NO_DEV_ERRORS='true' vite",
+    "build": "vite build",
```

Comment:
we don't have any tests anyway lol but if we start doing tests we can use [vitest](https://vitest.dev/)